src/Components/ViewArticles.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { newsApi } from "../App";
+import '../CSS/viewArticles.css'
+
+export default function ViewArticles () {
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState([false])
+ 
+    useEffect(() => {
+        setIsLoading(true)
+        newsApi("/articles").then((response) => {
```

Comment:
Please extract the axios instance and API call into a separate file (utils/api.js).